tests/test_runs.py

```diff
@@ -110,7 +110,7 @@ def test_tv_run_inversion(persistent_temp_model, monkeypatch):
 
     if beta_active:
 
-        fwd_beta = mdl_out.solvers[0].forward_beta
+        fwd_beta = mdl_out.solvers[0].forward
```

Comment:
No `[0]` here

Reply:
@jrmaddison if I take away this zero the test fails for the inversion ... 

the code below pass ... if not [0] fails ... 
fwd_beta = mdl_out.solvers[0].forward
        beta = mdl_out.solvers[0].beta

Reply:
it could be because the intention is to have multiple solvers at some point?